src/HotChocolate/Core/src/Types/Types/Pagination/PagingHelper.cs

```diff
@@ -33,25 +33,25 @@ public static class PagingHelper
             descriptor
                 .Use(placeholder)
                 .Extend()
-                .OnBeforeCreate((c, d) =>
+                .OnBeforeCreate((context, definition) =>
                 {
-                    MemberInfo? member = d.ResolverMember ?? d.Member;
-                    IExtendedType schemaType = GetSchemaType(c.TypeInspector, member, type);
-
-                    var configuration = new TypeConfiguration<ObjectFieldDefinition>
-                    {
-                        Definition = d,
-                        On = ApplyConfigurationOn.Completion,
-                        Configure = (c, d) => ApplyConfiguration(
-                            c, d, entityType, resolvePagingProvider, options, placeholder)
-                    };
-
-                    configuration.Dependencies.Add(
```

Comment:
This actually is correct sine we need the entity type to be named first. The issue as I pointed is that the field dependency is demanding completion.

Reply:
This is the issue

new TypeDependency(
                             TypeReference.Create(schemaType, TypeContext.Output),
                                                     TypeDependencyKind.Named));